paddle/operators/cond_op.cc

```diff
@@ -216,11 +216,12 @@ class CondOpProtoAndCheckerMaker : public framework::OpProtoAndCheckerMaker {
     AddOutput("IndexTensors", "Index Tensors contains indices for true/false");
 
     AddComment(R"DOC(
-Sample dependent Cond Operator:
-Given Cond[i] as a 1/0 vector to indicate true/false
-The equation is:
-Out[i] = subnet_t[i], if Cond[i] == true
-Out[i] = subnet_t[i], if Cond[i] == false
+Sample dependent Cond Operator.
```

Comment:
How do we decide about capitalization? Should this title be "Sample-dependent" or "Sample Dependent"?

Another question is should we say "Conditional Operator", or just keep it "Cond Operator"?

Reply:
I think we should go with all capitalization for this title. 
So I will change the title to "Sample Dependent Conditional Operator".

Regarding the "Conditional Operator" or "Cond Operator", I would say that if we know the meaningful full description of the Operator then we should go with the full name (like in this case, the author of this op already add "sample dependent" before cond). In other cases, we can go with the c++ class name (like "Cond", which should be a more common practice).